www/src/components/repository/RepositoryDescriptionMarkdown.js

```diff
@@ -0,0 +1,118 @@
+import React, { memo } from 'react'
+import { Markdown } from 'grommet'
+import {
+} from 'pluralsh-design-system'
+import { A, Blockquote, Box, Code, H1, H2, H3, H4, H5, H6, 
+  Li, Ol, Pre, Ul } from 'honorable'
+
+function MdImg({ src, gitUrl, ...props }) {
+  // Convert local image paths to full path on github
+  // Only works if primary git branch is named "master"
+  if (src && !src.match(/^https*/)) {
+    src = `${gitUrl}/raw/master/${src}`
+  }
+
+  return (
+    <img
+      src={src}
+      maxWidth="100%"
+      display="inline"
+      {...props}
+    />
+  )
+}
+
+function MdP({ ...props }) {
+  return <p {...props} />
+}
+
+function MdCode({ ...props }) {
+  return <Code {...props} />
```

Comment:
We do have the ability to syntax highlight here, if you look at one of the existing markdown components, you might be able to see how that was done previously (it uses highlight.js)

Reply:
here's a code pointer to it: https://github.com/pluralsh/plural/blob/master/www/src/components/incidents/Markdown.js#L77

Reply:
Also useful : www/src/components/utils/Code.js

Reply:
Now using www/src/components/utils/Code.js
I also refactored that a bit since it was adding a bunch of extra elements and padding around anything using syntax highlighting.

Reply:
Also involved implementing my own version of  `react-highlight.js` with hooks and `honorable` components.